### ISSUE TITLE
Function is not found initializeReflectable() and the file  'main.reflectable.dart' 

### ISSUE THREAD

**Issue by robertoltrocha** [eernstg/reflectable-flutter#5]
Hello,
I'm sorry for being annoying, but I'm having trouble installing this plugin, this import file is not found and initializeReflectable (); it is also not found. I tried several searches on the internet and got here. Could you tell me how I find or generate the file 'main.reflectable.dart' and how the system will find initializeReflectable ();

I downloaded your project and also had the same issue.

Thanks for your help.
![Capturar](https://user-images.githubusercontent.com/30963205/83718183-f0ea6b80-a60a-11ea-8960-fdd18d546ab5.PNG)

**Comment by eernstg** [comment]
This repository is just a tiny snippet of code that I created in order to confirm that reflectable works with Flutter, and also to show how it can be done.

However, reflectable is based on code generation, and you'll find more information about how to use it in general (with no references to Flutter) in the [reflectable repo](https://github.com/dart-lang/reflectable).

**Comment by robertoltrocha** [comment]
Hi,


I got it.

it works well.

Thank you so much.



Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Erik Ernst<mailto:notifications@github.com>
Enviado:quinta-feira, 4 de junho de 2020 15:12
Para: eernstg/reflectable-flutter<mailto:reflectable-flutter@noreply.github.com>
Cc:Roberto Rocha<mailto:rocha_roberto@hotmail.com>; State change<mailto:state_change@noreply.github.com>
Assunto: Re: [eernstg/reflectable-flutter] Function is not found initializeReflectable() and the file 'main.reflectable.dart' (#5)


This repository is just a tiny snippet of code that I created in order to confirm that reflectable works with Flutter, and also to show how it can be done.

However, reflectable is based on code generation, and you'll find more information about how to use it in general (with no references to Flutter) in the reflectable repo<https://github.com/dart-lang/reflectable>.
